public/template/list-client.pug

```diff
@@ -0,0 +1,12 @@
+.user.user-list
+  .row
+    .span12
+      include ./profile-block.pug
+
+  .row
+    span12
```

Comment:
I'll fix this in a followup so don't worry about it but this seems like a typo from the original utml -> Jade transition